roles/core/access_control/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
 - name: include_tasks ░ Use OS dedicated task
-  include_tasks: "{{ (ansible_facts.os_family|lower|replace(' ','_')) }}_{{ ansible_facts.distribution_major_version }}.yml"
+  include_tasks: "{{ (ansible_facts.os_family|replace(' ','_')) }}_{{ ansible_facts.distribution_major_version }}.yml"
```

Comment:
RHEL 7/8/9 file are identical:
```
$ sha256sum roles/core/access_control/tasks/RedHat_?.yml
4c04e6568865f6a4cb1da38f03fa1ddcf3a1adc02f8776f31000911f641f1bbe  roles/core/access_control/tasks/RedHat_7.yml
4c04e6568865f6a4cb1da38f03fa1ddcf3a1adc02f8776f31000911f641f1bbe  roles/core/access_control/tasks/RedHat_8.yml
4c04e6568865f6a4cb1da38f03fa1ddcf3a1adc02f8776f31000911f641f1bbe  roles/core/access_control/tasks/RedHat_9.yml
```

Can you use `with_first_found` to include `ansible_facts.os_family` and use RedHat.yml instead?